src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1173,7 +1190,7 @@ def _get_children_dependencies(extension_name):
     This function is making the reverse way : get all children that depend of a
     particular extension.
 
-    This is recurssive so this includes : children and so grand children and grand grand children
+    This is recursive so this includes : children and so grand children and grand grand children
```

Comment:
```suggestion
    This is recursive so this includes : children and so grand children and great grand children
```

If we are fixing for some reason in English we switch to great after the first grand. So then you just add more greats rather than more grands. Don't know know why :)